pkg/appfile/storage/storage.go

```diff
@@ -0,0 +1,52 @@
+package storage
+
+import (
+	"github.com/oam-dev/kubevela/pkg/appfile/storage/driver"
+)
+
+// Store application store client
+var Store *Storage
+
+func init() {
+	Store = NewStorage("")
```

Comment:
Using the default driver name `Local` instead of ""?

Reply:
Done